mlabs/test/Test/NFT/QuickCheck.hs

```diff
@@ -288,7 +288,7 @@ instance ContractModel NftModel where
     ActionInit {} -> do
       let hAdmin = h $ InitKey wAdmin
       callEndpoint @"app-init" hAdmin [toUserId wAdmin]
-      void $ Trace.waitNSlots 2
+      void $ Trace.waitNSlots 3
```

Comment:
You could export your `waitInit` and use it here as well from `mlabs/test/Test/NFT/Init.hs`, just so we have SSOT.

Reply:
Will do